lib/statsd/instrument/metric.rb

```diff
@@ -61,6 +61,9 @@ def initialize(options = {})
     @value       = options[:value] || default_value
     @sample_rate = options[:sample_rate] || StatsD.default_sample_rate
     @tags        = StatsD::Instrument::Metric.normalize_tags(options[:tags])
+    if StatsD.default_tags
+      @tags      = Array(@tags) | StatsD.default_tags
```

Comment:
`StatsD.default_tags` can be `nil`. `[:a] | nil` raises a TypeError. 

You probably have to wrap `default_tags` in `Array()`, or even better, do this in the setter above.

Reply:
The if statement before `@tags` is re-assigned should prevent this kind of exception to happen, right?

Reply:
ah yes, never mind me.